code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -37,12 +37,28 @@
 	var/seeking = FALSE
 	var/can_repair_constructs = FALSE
 	var/can_repair_self = FALSE
+	var/runetype
 
 /mob/living/simple_animal/hostile/construct/Initialize()
 	. = ..()
 	update_health_hud()
+	var/spellnum = 1
 	for(var/spell in construct_spells)
-		AddSpell(new spell(null))
+		var/the_spell = new spell(null)
+		AddSpell(the_spell)
+		var/obj/effect/proc_holder/spell/S = mob_spell_list[spellnum]
+		var/pos = 2+spellnum*31
+		if(construct_spells.len >= 4)
+			pos -= 31*(construct_spells.len - 4)
+		S.action.button.screen_loc = "6:[pos],4:-2"
+		S.action.button.moved = "6:[pos],4:-2"
+		spellnum++
+	if(runetype)
+		var/datum/action/innate/cult/create_rune/CR = new runetype(src)
+		CR.Grant(src)
+		var/pos = 2+spellnum*31
+		CR.button.screen_loc = "6:[pos],4:-2"

```

Comment:
NO.

Reply:
Its a design choice I made, a simple "I dont like the automatic button placement" would have sufficed. 

For constructs you can move them like any other button anyway. I just wanted a better default position. For the cultists? Maybe you can describe what you would want changed. During testing I was extremely annoyed because moving from talismans to spells meant you have gone from a slightly onerous yet manageable inventory-based system to a constantly swapping set of spells - it was way too tedious moving spells somewhere more convenient when you would just use them once and then they disappear, then you have to rinse and repeat for every single spell.

Reply:
Ofc, maybe make the dagger remember where you stuck the fucking buttons

Reply:
Item buttons suck and there's no solution with the current system. Any time you currently move an item with your hand it has to briefly put it into nullspace first, which means you "lose" the button for a split second and then regain it.

The only solution would be for me to make the button independent of an item and just have it search your inventory for a dagger to equip when you use it.